workflow_scripts/config.py

```diff
@@ -0,0 +1,18 @@
+# Originally there are some checker failures in weekly-CI
+# Skip them in test_model_zoo.py for now
+# TODO: fix these checker failures
+SKIP_CHECKER_MODELS = ['vision/classification/alexnet/model/bvlcalexnet-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/caffenet/model/caffenet-6.onnx',  # this should be caffenet-3.onnx; missing too
+                       'vision/classification/densenet-121/model/densenet-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/inception_and_googlenet/inception_v1/model/inception-v1-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/inception_and_googlenet/inception_v2/model/inception-v2-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/rcnn_ilsvrc13/model/rcnn-ilsvrc13-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/resnet/model/resnet50-caffe2-v1-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/shufflenet/model/shufflenet-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/squeezenet/model/squeezenet1.0-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/vgg/model/vgg16-7.onnx',  # only failed on Windows-CI
+                       'vision/classification/vgg/model/vgg19-7.onnx',  # only failed on Windows-CI
+                       'vision/classification/vgg/model/vgg16-bn-7.onnx',  # only failed on Windows-CI
+                       'vision/classification/vgg/model/vgg19-bn-7.onnx',  # only failed on Windows-CI
+                       'vision/classification/vgg/model/vgg19-caffe2-3.onnx',  # opset1 typeinference function missing
+                       'vision/classification/zfnet-512/model/zfnet512-3.onnx']  # opset1 typeinference function missing
```

Comment:
VGG related models only fail on Windows-CI (They can pass onnx.checker locally). In addition, other models fail because some of operators of opset3 do not support `TypeAndShapeInferenceFunction` for shape inference. Have sent a PR to deprecate those olde models using opset3 which cannot pass onnx.checker:
https://github.com/onnx/models/pull/389

Reply:
They are failing because loading too many large ONNX models will cause MemoryError on Windows... To workaround this, move the weekly-CI from Windows to Linux.